plugins/woocommerce-admin/client/wp-admin-scripts/woo-connect-notice/index.js

```diff
@@ -0,0 +1,32 @@
+window.jQuery( document ).ready( function () {
```

Comment:
Just a note that there are some old discussions (i..e https://github.com/woocommerce/woocommerce/issues/18395) to remove jQuery as dependency from WC.